com.sap.cloud.lm.sl.common/src/main/java/com/sap/cloud/lm/sl/common/util/MapUtil.java

```diff
@@ -54,4 +57,18 @@ private MapUtil() {
         }
         return result;
     }
+    
+    public static boolean parseBooleanFlag(final Map<String, Object> parameters, String flagName, boolean defaultValue) throws ContentException {
```

Comment:
Can we make a generic function to accept all possilbe types, such as:
com.sap.cloud.lm.sl.cf.core.configuration.Environment::getVariable(String name, Function<String, T> mappingFunction, T defaultValue)

If you consider is as overengineering leave it as it is.